django/db/backends/mysql/creation.py

```diff
@@ -11,3 +16,39 @@ def sql_table_creation_suffix(self):
         if test_settings['COLLATION']:
             suffix.append('COLLATE %s' % test_settings['COLLATION'])
         return ' '.join(suffix)
+
+    def _clone_test_db(self, number, verbosity, keepdb=False):
+        qn = self.connection.ops.quote_name
```

Comment:
Thanks for point out that feature. I hadn't noticed it before.  Looks like it changed a bit with 1.7, though